spec/asyncapi.md

```diff
@@ -710,8 +710,8 @@ Field Name | Type | Description
 <a name="operationObjectSummary"></a>summary | `string` | A short summary of what the operation is about.
 <a name="operationObjectDescription"></a>description | `string` | A verbose explanation of the operation. [CommonMark syntax](http://spec.commonmark.org/) can be used for rich text representation.
 <a name="operationObjectSecurity"></a>security | [[Security Requirement Object](#securityRequirementObject)]| A declaration of which security mechanisms are associated with this operation. Only one of the security requirement objects MUST be satisfied to authorize an operation. In cases where Server Security also applies, it MUST also be satisfied.
-<a name="operationObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for API documentation control. Tags can be used for logical grouping of operations.
-<a name="operationObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this operation.
+<a name="operationObjectTags"></a>tags | [[Tags Object](#tagsObject) \| [Reference Object](#referenceObject)] | A list of tags for API documentation control. Tags can be used for logical grouping of operations.
```

Comment:
super tiny nit, but do we need the extra square brackets around the type? 
(as you didn't do that for External Documentation)

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/1444788/181749630-a1f9ea7e-3513-4ea5-a04a-8b960e78b959.png">